src/modules/0.0.18/aws_ecs_simplified/entity/ecs_simplified.ts

```diff
@@ -47,4 +47,15 @@ export class EcsSimplified {
 
   @Column({ nullable: true, })
   loadBalancerDns?: string;
+
+  @Column({
+    default: false,
+  })
```

Comment:
I'm actually thinking we shouldn't expose this and just always set it to `true` in `service`